src/filters/webgl_backend.class.js

```diff
@@ -2,6 +2,24 @@
 
   'use strict';
 
+
+  /**
+   * Tests if webgl supports certain precision
+   * @param {WebGL} Canvas WebGL context to test on
+   * @param {String} Precision to test can be any of following: 'lowp', 'mediump', 'highp'
+   * @returns {Boolean} Whether the user's browser WebGL supports given precision.
+   */
+  function testPrecision(gl, precision){
+    var fragmentSource = 'precision ' + precision + ' float;\nvoid main(){}';
+    var fragmentShader = gl.createShader(gl.FRAGMENT_SHADER);
+    gl.shaderSource(fragmentShader, fragmentSource);
+    gl.compileShader(fragmentShader);        
+    if (!gl.getShaderParameter(fragmentShader, gl.COMPILE_STATUS)) {
+      return false;
+    }
+    return true;
+  }
+
```

Comment:
unsure about the position of this function.
fabric.isWebglSupported  can be overridden by a dev with something more fancy.

testPrecision is here with a jsdoc, but will not be catched is not attached to any object, and can't be used from outside or from an overridden isWebglSupported. can you think of something better than calling it fabric.testPrecision?

does not block merging

Reply:
Sorry, I don't follow. Do you want to move testPrecision to fabric object as well?
Like all developers, I am not good at naming things :D

Reply:
Maybe I should move the testprecision function inside isWebGlSupported function?

Reply:
no i would leave it here. i was just wondering if it makes sense to have a non accessible function that can't be reused if a dev change the implementation of `isWebglSupported`. not a problem at all.